src/bootstrap/src/core/build_steps/test.rs

```diff
@@ -1773,7 +1779,7 @@ NOTE: if you're sure you want to do this, please open an issue as to why. In the
                 target,
             });
         }
-        if suite == "run-make" {
+        if mode == "run-make" {
```

Comment:
Remark: I almost missed this until I noticed that the snapshot tests for `./x test run-make-cargo` didn't have a step for building `run-make-support`, and I was like "huh, how strange".